cmd/webhook/main.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"context"
+	"os"
+
+	"github.com/openshift-pipelines/manual-approval-gate/pkg/reconciler/webhook"
+	"knative.dev/pkg/configmap"
+	"knative.dev/pkg/controller"
+	"knative.dev/pkg/injection"
+	"knative.dev/pkg/injection/sharedmain"
+	"knative.dev/pkg/signals"
+	kwebhook "knative.dev/pkg/webhook"
+	"knative.dev/pkg/webhook/certificates"
+)
+
+func newValidationAdmissionController(name string) func(ctx context.Context, cmw configmap.Watcher) *controller.Impl {
+	return func(ctx context.Context, cmw configmap.Watcher) *controller.Impl {
+		return webhook.NewAdmissionController(ctx,
+			name,
+			"/approval-validation",
+			func(ctx context.Context) context.Context {
+				return ctx
+			},
+			true,
+		)
+	}
+}
+
+func main() {
+	serviceName := os.Getenv("WEBHOOK_SERVICE_NAME")
```

Comment:
@PuneetPunamiya are we using these environment variables only here?
`WEBHOOK_SERVICE_NAME`
`WEBHOOK_SECRET_NAME`
`WEBHOOK_ADMISSION_CONTROLLER_NAME`

If we use it on different places, please create a constant

Reply:
No we are not using it in different places